src/stories/Blocks/pages/pages.scss

```
@@ -0,0 +1,56 @@
+%hero-2-column {
+  @include layout-container($layout__max-width--large);
+  border-bottom: 1px solid $color__global-tertiary-1;
+  display: grid;
+  gap: $s-lg;
+  padding-bottom: $s-2xl;
+  @include media-query__small {
+    grid-template-columns: 2fr 1fr;
+    gap: 0;
+    padding-bottom: 0;
+  }
+
+  // if hero-content is a link, remove text-decoration for all children
+  > a {
+    text-decoration: none;
+  }
+
+  .hero-content,
+  .hero-visual {
+    @include media-query__small {
+      padding: $s-2xl;
+    }
+  }
+
+  .hero-content {
+    padding-top: $s-md;
+
+    @include media-query__small {
+      padding-top: $s-3xl;
+    }
+
+    // Add paddig on childs except .hero-cta
+    > *:not(.hero-cta) {
```

I'm not a fan of using `:not` when it can be avoided.

Couldnt you just do:

```suggestion
    > * {
            padding: 0 $s-3xl;
      @include media-query__small {
        padding: 0;
      }
    }
    
    >.hero-cta {
     padding: 0;
    }
```